db/manual_compaction_test.cc

```diff
@@ -13,7 +13,21 @@
 #include "rocksdb/write_batch.h"
 #include "test_util/testharness.h"
 
-namespace ROCKSDB_NAMESPACE {
+using ROCKSDB_NAMESPACE::CompactionFilter;
+using ROCKSDB_NAMESPACE::CompactionStyle;
+using ROCKSDB_NAMESPACE::CompactRangeOptions;
+using ROCKSDB_NAMESPACE::DB;
+using ROCKSDB_NAMESPACE::DestroyDB;
+using ROCKSDB_NAMESPACE::FlushOptions;
+using ROCKSDB_NAMESPACE::Iterator;
+using ROCKSDB_NAMESPACE::kNoCompression;
```

Comment:
This should probably be CompressionType, not kNoCompression

Reply:
I believe that's only relevant for `enum class` not legacy `enum`.

From https://en.cppreference.com/w/cpp/language/using_declaration
> Only the name explicitly mentioned in the using-declaration is transferred into the declarative scope: in particular, enumerators are not transferred when the enumeration type name is using-declared.

Reply:
Looks like `using ROCKSDB_NAMESPACE::CompressionType` is better, and I need to update two assignments to
`... = CompressionType::kNoCompression`.